### PR TITLE
add `skip-paranoid-checks` flag for tikv-ctl

### PR DESCRIPTION
**Pull request by hicqu**
Need to use `pingcap/rust-rocksdb`.

### CODE REVIEW

File: src/bin/tikv-ctl.rs [hunk 1]
```diff
@@ -57,6 +57,7 @@ fn perror_and_exit<E: Error>(prefix: &str, e: E) -> ! {
 fn new_debug_executor(
     db: Option<&str>,
     raft_db: Option<&str>,
+    skip_paranoid_checks: bool,
```
Comment (siddontang) [inline src/bin/tikv-ctl.rs]: oh, If we will another config later, do we need to add a new param here again? 

why not using this in config directly?

Comment (hicqu) [reply src/bin/tikv-ctl.rs]: You mean add it in `src/config.rs`? I think only `tikv-ctl` needs it, `tikv-server` should never use the flag. So I prefer to leave it in `debug.rs`.

**Comment by hicqu** [comment]
PTAL @Connor1996 @zhangjinpeng1987

**Pull request merged**